app/models/movie.rb

```diff
@@ -126,8 +101,8 @@ def priority(list)
   end
 
   def priority_text(list)
-    p = listings.find_by(list_id: list.id).priority
-    case p
+    priority = listings.find_by(list_id: list.id)&.priority
```

Comment:
`p` feels reserved to me. If we ever wanted to do any debugging in this method and wanted to output the value of `p`, we may have a collision, so i renamed it to `priority`.

Reply:
👍 Totes